paddle/fluid/framework/dlpack_tensor.cc

```diff
@@ -77,32 +73,32 @@ static DLDataType GetDLDataTypeFromTypeIndex(proto::VarType::Type type) {
 #undef REG_DL_DATA_TYPE
 }
 
-struct DLContextVisitor : public boost::static_visitor<::DLContext> {
```

Comment:
Why `Context` -> `Device`?

Reply:
Since we update DLpack version from 0.2 to 0.4, we need to change `context` to `device` following the update version.